public/js/orders/shopping-cart.js

```diff
@@ -1,9 +1,13 @@
+const SHOPPINGCART_COOKIE_NAME      = 'shopping-cart';
+const SHOPPINGCART_COOKIE_DOMAIN    = '127.0.0.1';
```

Comment:
Ik vraag me af of dit wel werkt als de link niet meer local host is, dus als het gehost staat op azure, zou het kunnen dat dit dan crasht

Reply:
Ik had het in een cookie.js gezet, maar om die te include moet de includer een module worden waardoor button clicks niet meer gebruikt kunnen worden zonder events waardoor het niet meer zou werken. Dit moet idd in de Deployment handleiding komen.

Reply:
Dat lijkt mij niet zo handig. Laravel heeft ook gewoon een manier om met cookies om te gaan. Waarom gebruik je dat niet?

Reply:
Oh ik denk zelf dat dat toch beter is dan hoe het nu gaat.

Reply:
Dat is wel een wate grotere refactor

Reply:
Dat is waar. Je kan het ook oplossen door de current url gewoon op te halen? https://ui.dev/get-current-url-javascript